besu-operator.md

```diff
@@ -0,0 +1,34 @@
+# Lab Name
+Besu Operator
+
+# Short Description
+Kubernetes operator to provide an easier way of packaging, deploying, and managing Hyperledger Besu network.
+
+# Scope of Lab
+Deploying applications using plain Docker containers has many issues such as :
+
+1. No management layer on top of it
+   - If a container dies, nothing will restart it
+   - If a virtual machine crashes, no failure recovery
+2. No monitoring, alerts, etc.
+3. Not scalable, no easy way of spreading containers among VM’s
+
+The solution to all these is the Kubernetes, which undoubtedly is the future of container orchestration. Kubernetes (also known as k8s or “kube”) is an open source container orchestration platform that automates many of the manual processes involved in deploying, managing, and scaling containerized applications. It has a large, rapidly growing ecosystem. Kubernetes services, support, and tools are widely available.
```

Comment:
Change `the Kubernetes` to `Kubernetes`

Reply:
One could argue that Kubernetes is not "the future of container orchestration". It already is the present! ;-)
Joke apart, I think it's kind of subjective statement isn't necessary. I would rephrase: "A solution to all these is Kubernetes." and proceed with the next sentence "Kubernetes (also known..."

Reply:
@tkuhrt Thanks for the suggestions, I have made the changes.